tests/ToastMessage.elm

```diff
@@ -15,4 +15,17 @@ suite =
                 Toast.encode toast
                     |> decodeValue Toast.decoder
                     |> Expect.equal (Ok toast)
+        , fuzz v1Fuzzer "Can decode old messages" <|
+            \v1Toast ->
+                Fixture.encodeV1 v1Toast
+                    |> decodeValue Toast.decoder
+                    |> Expect.equal
+                        (Ok
+                            (Toast
+                                (v1Toast.title |> Maybe.withDefault "Notification!")
+                                v1Toast.message
+                                v1Toast.custom
+                                "info"
```

Comment:
To validate old versions, we generate a bunch randomly, decode them, and verify that the input data shows up correctly in the generated `Toast` object, or that the correct defaults are used.